src/Apps/__stories__/Comparables.story.tsx

```diff
@@ -0,0 +1,15 @@
+import { Box } from "@artsy/palette"
+import { storiesOf } from "@storybook/react"
```

Comment:
should be `import { storiesOf } from "storybook/storiesOf"` so that it pulls in theme, boot, etc.